packages/shared/examples/examples-list.tsx

```diff
@@ -25,6 +25,13 @@ export const examples: ExampleCollection[] = [
       require('./horizontal-stacked-bar-chart').default,
     ],
   },
+  {
+    title: 'Donut Charts',
```

Comment:
Instead of making a new category, I would add it as the first entry to "Circular Charts"